misc/formatter/modules/formatter-core/src/test/resources/misc/linewrapping/assert/line_wrapping_1.bal

```diff
@@ -0,0 +1,21 @@
+function foo(int count, int count, int count, int count, int count, int count, int count, int count, 
+             int count = 999999999) {
+}
+
+public type Client client object {
+    remote function query(@untainted string|ParameterizedQuery sqlQuery, typedesc<record {}>? rowType = ())
+    returns @tainted stream<record {};, Error>;
+}
+
+function foo() {
+    ReportTable|error reportTable = table key(id) from var student in studentList
+        where student.gpa >= 2.0
+        let string degreeName = "Bachelor of Medicine", int graduationYear = calGraduationYear(student.intakeYear), int test = 
+        calGraduationYear(student.intakeYear)
+        select {id: student.id};
+}
+
+public function main() {
+    string stringVal = 
+    "this is a sample. this is a sample. this is a sample. this is a sample. this is a sample. this is a sample. " + " this is a sample. this is a sample. this is a sample." + "this is a sample. this is a sample. this is a sample. this is a sample. this is a sample. this is a sample. this is a sample." + " this is a sample. this is a sample.";
```

Comment:
In this result, shouldn't the formatted source look like follows? (with multiple wrappings)

```ballerina
string stringVal = 
    "this is a sample. this is a sample. this is a sample. this is a sample. this is a sample. this is a sample. " + 
    " this is a sample. this is a sample. this is a sample." + 
    "this is a sample. this is a sample. this is a sample. this is a sample. this is a sample. this is a sample. this is a sample." + 
    " this is a sample. this is a sample.";
```

Reply:
Yes, will try to incorporate this improvement as well.